stories/automilos.fmf

```diff
@@ -0,0 +1,15 @@
+story:
+    As AutoMiloš I want to execute relevant tests when the issue is switched to given state
+
+description:
+    Test/plan run on AutoMilos should be triggered when the issue is switched to given state.
```

Comment:
story calls it  `AutoMiloš`, description `AutoMilos` (just 's' not 's with caron')